olp-cpp-sdk-authentication/src/AuthenticationClient.cpp

```diff
@@ -659,6 +626,39 @@ std::string AuthenticationClient::Impl::base64Encode(
   return ret;
 }
 
+ServerTimeResponse AuthenticationClient::Impl::GetServerTime(
+    std::string server_url, const std::shared_ptr<http::Network>& network,
```

Comment:
```suggestion
   std::string server_url, http::Network& network,
```
No need for shared_ptr reference.